dev/com.ibm.websphere.appserver.thirdparty.eclipselink.2.7/resources/org/eclipse/persistence/internal/localization/i18n/ToStringLocalizationResource.nlsprops

```diff
@@ -0,0 +1,179 @@
+###############################################################################
```

Comment:
General comment about all of these .nlsprop files... Are these 100% duplicated from JPA 2.1?  If so, can we possibly share them?  It would be nice to not have to worry about dual-maintenance, and having one set of these nlsprop files would reduce the overall product size on disk a bit

Reply:
Right now, they're duplicated from Eclipselink 2.6, I haven't run the tool yet to collect new NLS messages that were added since 2.6_WAS had split from master on Eclipselink's git repository.  That will be a followup item.

Reply:
Almost all of them are duplicates from JPA 2.1/Eclipselink 2.6.  However 3 NLS props files needed to be updated, see my attachment below.